docs/contributing/screenshot-testing.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Screenshot Testing
+---
+
+## What is the Screenshot Testing
+Screenshot test is visual test that takes a screenshot, then compares it to a reference screenshot file saved. When ui changes, one of the major considerations is whether visual changes exist. Some changes may be easy to check, but some changes are hard to check due to very small change. With Screenshot test, very small changes can easily check due to compare pixel-by-pixel. Additionally, Screenshot testing is the most reliable test for static images.
+
+## Prerequisites
+Clone sandstone from GitHub, install dependencies and connect the modules using Lerna:
+
+```shell
+# clone the repo!
+git clone git@github.com:enactjs/sandstone.git
+# move in
+cd sandstone
+# we're using git flow so develop is our working branch
+git checkout develop
+# install lerna
+npm install
+# link dependencies
+enact link
+```
+
+## Creating tests
+Within the UI Library, create an app for testing in `./tests/screenshot/apps` and create a corresponding test in `./tests/screenshot/specs`.
+
+	+ src
+		+ test
+			+ screenshot
+				+ apps
+					+ component
+						+ testComponent
+							testComponent.js		<-- create app here
+				+ specs
+						testComponent-specs.js		<-- create spec file here
+
+In screenshot test, create apps to test components. Please refer sample code.
+* Button.js
+
+```JS
+import Button from '../../../../Button';
+
+const ButtonTests = [
+	<Button> Hello <Button>
+];
+
+export default ButtonTests;
+```
+And create spec to test additional specification. It is commonly used by all components. Please refer sample code.
+* Common-spec.js
+
+```JS
+const {Page} = require('@enact/ui-test-utils/utils');
+const {runTest} = require('@enact/ui-test-utils/utils');
+
+runTest({
+	testName: 'Common',
+	Page: Page,
+	skin: 'light'
+});
+```
+The generateTestData and Page component from `@enact/ui-test-utils/utils/` generateTestData and Page contain useful methods for screenshot tests.
```

Comment:
```suggestion
The `generateTestData` and `Page` component from `@enact/ui-test-utils/utils/` generateTestData and Page contain useful methods for screenshot tests.
```
But I'm not sure if this part is really necessary.

Reply:
I removed this part and added link for ui-test-utils.